README.md

```diff
@@ -1,205 +1,82 @@
-
-```bash
-curl 'http://127.0.0.1:7700/indexes/movies/search?q=botman+robin&limit=2' | jq
-```
-
-```json
-{
-  "hits": [
-    {
-      "id": "415",
-      "title": "Batman & Robin",
-      "poster": "https://image.tmdb.org/t/p/w1280/79AYCcxw3kSKbhGpx1LiqaCAbwo.jpg",
-      "overview": "Along with crime-fighting partner Robin and new recruit Batgirl, Batman battles the dual threat of frosty genius Mr. Freeze and homicidal horticulturalist Poison Ivy. Freeze plans to put Gotham City on ice, while Ivy tries to drive a wedge between the dynamic duo.",
-      "release_date": 866768400
-    },
-    {
-      "id": "411736",
-      "title": "Batman: Return of the Caped Crusaders",
-      "poster": "https://image.tmdb.org/t/p/w1280/GW3IyMW5Xgl0cgCN8wu96IlNpD.jpg",
-      "overview": "Adam West and Burt Ward returns to their iconic roles of Batman and Robin. Featuring the voices of Adam West, Burt Ward, and Julie Newmar, the film sees the superheroes going up against classic villains like The Joker, The Riddler, The Penguin and Catwoman, both in Gotham City… and in space.",
-      "release_date": 1475888400
-    }
-  ],
-  "nbHits": 8,
-  "exhaustiveNbHits": false,
-  "query": "botman robin",
-  "limit": 2,
-  "offset": 0,
-  "processingTimeMs": 2
-}
-```
-
-#### Use the Web Interface
-
-We also deliver an **out-of-the-box [web interface](https://github.com/meilisearch/mini-dashboard)** in which you can test Meilisearch interactively.
-
-You can access the web interface in your web browser at the root of the server. The default URL is [http://127.0.0.1:7700](http://127.0.0.1:7700). All you need to do is open your web browser and enter Meilisearch’s address to visit it. This will lead you to a web page with a search bar that will allow you to search in the selected index.
 
-| [See the gif above](#demo)
+- **Search-as-you-type experience** — Provides relevant search results in less than 50 milliseconds.
+- **Typo tolerant** — Understands misspellings and adapts to return relevant results.
+- **Facetting and filtering** — Enhance your user's search experience with custom filters and build a faceted search interface in a few lines of code.
+- **Sort by** — Sort results based on price, date, or pretty much anything else your users need.
+- **Synonyms** — Meilisearch supports synonyms, so each of your users will find the content they are looking for.
+- **Geosearch** — Filter, sort documents around real-world geographic data.
+- **Language support** — Meilisearch supports all languages and detects them automatically.
+- **Secure** — **Secure your data with API keys that allow fine-grained permissions management.
+- **Multi-Tenant** — Personalize the search results for each of your application's tenants.
+- **Highly Customizable** — Meilisearch provides presets that work out of the box but are customizable for your specific needs.
+- **RESTful API** — Integrate seamlessly into your technical stack with our SDKs/Integrations.
```

Comment:
It's not true because of the asynchronous? Because apart from that we follow the Rest conventions as best as we can so far, especially with v0.28

Reply:
Yes, I can also understand what you mean! I let the discussion open in case someone (like @guimachiavelli or @CaroFG) has something to add.

Reply:
RESTful API is also highlighted in [the documentation](https://docs.meilisearch.com/learn/what_is_meilisearch/overview.html), if it's not true, we may want to change it...

Reply:
TL;DR
If all we want here is to focus on how easy it is to create plugins and integrations, I would recommend removing RESTful API. It's simpler and safer, IMHO.

```suggestion
- **Seamless integration** — use our SDKs or create your own tools to add Meilisearch to your stack
```

I'm not fully aware of everything that goes into making an API RESTful, so I don't feel informed enough to vote yes or no on this matter. I think the most important question is: will a user who blindly trusts us when we say we are RESTful get in trouble? Like, might they start using Meilisearch and then realise they cannot do something because we are not fully RESTful?

If the answer to those questions is yes, then we should absolutely remove it, as otherwise users would be justified in thinking we are lying to them. Getting someone's trust back after they feel they have been lied to is really difficult.

Reply:
We definitely offer a [RESTFul API](https://www.techtarget.com/searchapparchitecture/definition/RESTful-API) in its philosophy. In my opinion, we can totally keep this term and put it forward.

Regarding the readme itself, I think it's good to have the term within the explanation sentence while keeping seamless integration as the bolded text.

Reply:
I suggest keeping RESTful API as an important feature to highlight! The first proposal does the job imo, we can always update it in the future, it's not a breaking change. 😗